backend/models/transaction.py

```
@@ -120,12 +285,46 @@ async def update_transaction_with_event_info(transaction_id: str, event_id: str,
             logging.error(f"Error updating transaction {transaction_id} with event info: {e}")
             return False
     
+    @staticmethod
+    async def update_transaction_refund_info(transaction_id: str, refund_type: str, refunded_on: str = None, refunded_amount: Decimal = None):
```

Ruff (0.14.2)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)